### DISCUSSION TITLE
How can edit the maximum allowed 10 MB

### DISCUSSION THREAD

**Discussion by kevinzDzul** [spatie/laravel-medialibrary#2325] [Q&A]
Hi, I am trying to update the maximum limit to move the files from 10M to 100M. I have edited the media-library file but no change happens, it remains at 10M, what can I do.

in media-library i change to
`
...
  'max_file_size' => 1024 * 1024 * 100 
...
`

**Comment by squatto** [comment]
You will also need to make sure that PHP itself has an increased upload limit. How you accomplish this is going to depend on your environment, but if you have the ability to change your `php.ini` settings, [here is a good tutorial for how to do it](https://www.a2hosting.com/kb/developer-corner/php/using-php.ini-directives/php-maximum-upload-file-size?aid=1656214).
